integrated-developer-portal/apiproxy/sample-integrated-developer-portal.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2022 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<APIProxy revision="1" name="sample-integrated-developer-portal">
```

Comment:
We dont want all these to be checked in, just replace the file with
```xml
<APIProxy name="sample-integrated-developer-portal">
  <Description>An API key protected proxy for an integrated developer portal</Description>
</APIProxy>
```

Reply:
Can you explain the reasoning on this one? What do you mean by "checked in"?

Reply:
I meant committing to the repo.. just have the lines I pasted above